### ISSUE TITLE
Load image node inaccuracy with the alpha channel.

### ISSUE THREAD

**Issue by Kiaazad** [comfyanonymous/ComfyUI#1515]
The load image node fills the alpha channel with black, but it looks like the process is very inaccurate.
![image](https://github.com/comfyanonymous/ComfyUI/assets/33455007/18c2ca01-b25c-4992-88e9-c7ff26319b26)
Can we get a bit more accuracy, and perhaps the option to choose the fill color please?

**Comment by Kiaazad** [comment]
Another issue I have with the image loader is trying to constrain the image and it's mask, I have to convert the mask into image, and use `pyssss constrain image` node, the convert it back to mask because the image loader removes the alpha channel and I can't get the mask after constraining it.
It would be much easier if we could do all kind of manipulation on the image with it's alpha channel intact, then fill the background with a node before sending it to the encoder.

**Comment by MoonRide303** [comment]
@Kiaazad Would that kind of nodes help (#1577)? You should be able to do any kind of composition you want with it (when it's merged).
![image](https://github.com/comfyanonymous/ComfyUI/assets/130458190/6a62b0bb-2df6-46ff-8579-16758008ed55)

**Comment by jn-jairo** [comment]
> @Kiaazad Would that kind of nodes help (#1577)? You should be able to do any kind of composition you want with it (when it's merged). ![image](https://user-images.githubusercontent.com/130458190/270105123-6a62b0bb-2df6-46ff-8579-16758008ed55.png)

@MoonRide303 do you mean this?

![image](https://github.com/comfyanonymous/ComfyUI/assets/5104869/32c70225-796e-44da-a38d-6659b2cb0a33)

You can do it with the current nodes.

**Comment by MoonRide303** [comment]
@jn-jairo You're right - the simplest modes are possible with basic compostite, too.

**Comment by jn-jairo** [comment]
@Kiaazad I found the problem, check it out:

![image](https://github.com/comfyanonymous/ComfyUI/assets/5104869/9ec82640-9d98-4845-b2a8-744e4f300395)

That is a problem in how image editors stores the data in the channels, see the curved line in the center of the image, I tried the brushes and then I erased it and draw the straight lines, but the software kept this line in the RGB channels and just updated the Alpha channel, also the straight lines I made have some blur around it but that was stored only in the Alpha channel.

The same thing happened in your case, the image you see in the preview is the image the software you used to create that image stored in the RGB channel, exactly like that, those thin lines only exist in the Alpha channel of your image.

**Comment by Kiaazad** [comment]
I see. That must be the cause of artifacts I've been getting around edges of my mask.
I think with a couple of nodes, we should be able to prune the pixels that are saved with wrong data under the alpha mask. I'll string a couple of nodes together and see what I can come up with.
Maybe an option or node to prune that data would be a good enhancement, although, precise noise masking must be a niche use case if I'm the first one that noticed it.
Thank you everybody
